remote-ingestion-executor/variables.tf

```diff
@@ -10,6 +10,8 @@ variable "datahub" {
     executor_id = optional(string, "remote")
     # SQS Queue ARN
     queue_url = string
+    # SQS Queue aws region
+    aws_region = optional(string, data.aws_region.current.name)
```

Comment:
@dim-ops I don't think `data` is allowed here? I've submitted #9 as an alternative.

Reply:
Good point, I totally agree